src/blocks/product-card.css

```diff
@@ -0,0 +1,125 @@
+:root {
+  --color-aquamarine: #00acdc;
+  --color-dark-white: #f3f3f3;
+  --color-navy-black: #060b35;
+  --color-light-brown: #616070;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  background: white;
+  border: 1px solid var(--color-dark-white);
+  border-radius: 5px;
+  width: 200px;
+  font-family: Roboto, sans-serif;
+  padding: 0 16px;
+  box-sizing: border-box;
+}
+
+.product-card__image {
+  padding: 0 3px;
+  width: 160px;
+  height: 134px;
+  padding-top: 32px;
+  margin-bottom: 40px;
+}
+
+.product-card__name {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-navy-black);
+  height: 36px;
+  margin-bottom: 4px;
+  margin-top: 0;
+}
+
+.product-card__code {
+  width: 99px;
+  height: 14px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-light-brown);
+  margin-bottom: 16px;
+  margin-top: 0;
+}
+
+.product-card__review {
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+  height: 16px;
```

Comment:
_[STYLES] - Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size._

Reply:
I think this refers to block. Not for elements. I deleted `height`, because is useless.